### DISCUSSION TITLE
Asside Info button/link for non-admins (href="/admin/docs")

### DISCUSSION THREAD

**Discussion by kzsa** [directus/directus#7748] [Feature Requests]
Hi,
I suggest you make it possible to change the visibility and the purpose of the Info button/link on the asside for non-administrator users.
Now the info button contains relevant information for the admin users only. On a customized system the non-admin users might expect something else when clicking the button. (They may expect brainstorming from the administrator, who configured the system.)
I mean, an information button with instructions from the sysadmins (in the user's native language) would be more useful. The original documentation can be disappointing for them, because it is too detailed and contains such as options over which only the administrator has control.
TFYA

Br: kzsa

**Comment by benhaynes** [comment]
Are you referencing this info area? Which pages are you referring to?

![Screen Shot 2021-09-01 at 11 10 07 AM](https://user-images.githubusercontent.com/522079/131696781-4172e070-a513-450e-940f-39417828dee1.png)

I agree that it would be nice if we allowed admins to customize the text in this area. We could use the Collection Description here to give more control over editing.

**Reply by kzsa** [reply]
Hi,
On both sides as follows?

Admin:
Left Directus info button as it is now. (I think it's cool as it is!)
The right one shown by you is editable. (Admins among themselves / tricks / look at it / I hacked it / etc.)

For content editor users:
Hide left Directus info button.
On the right side, shown by you is editable as admin. (Localised help tailored to "plain" users / content editors)

Br: kzsa

**Reply by benhaynes** [reply]
Ahh, you're talking about hiding the "Docs" module on the left. That can already be done using the Module Override within Settings > Roles. 👍

**Reply by kzsa** [reply]
Thanks for the info! The more I play with it, the more I like it!
...and your communication is exemplary. I've never seen that with any team. Where does all this energy come from? Congratulations to both your product and your team. It is clear that you want to create value. 👏 🤝

Br: Zsolt

**Reply by benhaynes** [reply]
Haha, thanks! We're always excited to share and improve Directus... plenty of energy for that! ⚡

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
